modules/foreman_bookmark.py

```diff
@@ -156,17 +146,19 @@ def main():
         supports_check_mode=True,
     )
 
-    (bookmark_dict, state) = module.parse_params()
+    (entity_dict, state) = module.parse_params()
 
     module.connect()
 
-    entity = find_bookmark(
-        module, bookmark_dict['name'], bookmark_dict['controller'], failsafe=True)
+    try:
+        search = 'name="{}",controller="{}"'.format(entity_dict['name'], entity_dict['controller'])
+        entity = module.find_resource('bookmarks', search, failsafe=True)
+    except Exception as e:
```

Comment:
Heads up, we are about to be getting rid of that try block:
https://github.com/theforeman/foreman-ansible-modules/pull/281

Reply:
Ok, maybe I'll wait for that one to be merged then. Guess it would be best to get #279 in as well. 🙂

Reply:
Well, that one is hard to predict...

Reply:
I wouldn't wait on #279 -- that can take some time, and is "easy" as it's just a `git mv`.

#281 is merged tho :)

Reply:
Cool! Removing the try-catch then.